woocommerce/woocommerce
# Update Action Scheduler to 3.8.2

### Changes proposed in this Pull Request:

Updates the Action Scheduler package to the latest release.

### How to test the changes in this Pull Request:

In the Admin UI of your test site, go to WooCommerce > Status, scroll down to the Action Scheduler section, and make sure the version shown is 3.8.2.
